elasticsearch-extensions.php

```diff
@@ -15,6 +15,9 @@
 use Elasticsearch_Extensions\Controller;
 use Elasticsearch_Extensions\Registry;
 
+// Load Composer dependencies.
+require_once __DIR__ . '/vendor/wordpress-autoload.php';
```

Comment:
I'm not sure how people are adding/installing this plugin but I'd assume this change would break their sites (in case they pull directly from GitHub).

Reply:
How would it break their sites? Because they need to run `composer install` on it?

It should be installed as a plugin via composer, although it could be submoduled, but in any case, needing to run `composer install` on it is fine.

Reply:
Cool! Thanks for the confirmation.

But yes, my concern was a site could be using the plugin without using `composer`. Pulling the latest would cause a fatal error since the `vendor` folder would not exist.

Reply:
If we ever get around to publishing this on WordPress.org we will add a step that installs the vendor folder and includes that in what is sent to the WP.org SVN repo (as well as removing some development files that wouldn't be needed in a production install). But for now it's fine to leave as-is. You might want to add a note in the README about needing to run `composer install` if that's not there already.